yeastphenome/apps/conditions/models.py

```diff
@@ -130,9 +122,7 @@ def papers(self):
         )
 
     def papers_link_list(self):
```

Comment:
Same for this one - let's make sure if it's not used and not needed we remove, otherwise add to where it's supposed to be.